flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/DashboardConfiguration.java

```diff
@@ -139,6 +146,11 @@ public boolean isWebCancelEnabled() {
             return webCancelEnabled;
         }
 
+        @JsonIgnore
+        public boolean isWebHistoryEnabled() {
```

Comment:
I think the name is misleading. Unlike `isWebSubmitEnabled` and `isWebCancelEnabled` which indicate whether a feature is enabled, this indicates whether the dashboard is served by a running cluster or a history server. I'd suggest the name `isHistoryServer`.

Same for the relevant field/variable/method names.